content/docs/reference-react-component.md

```diff
@@ -413,6 +413,12 @@ class ErrorBoundary extends React.Component {
 }
 ```
 
+Production and development builds of React slightly differ in the way `componentDidCatch()` handles errors.
```

Comment:
wasted 2 hours trying to debug this... 💀 

just realised it was all working fine